projectq/meta/_compute.py

```diff
@@ -191,32 +206,7 @@ def add_uncompute(command, old_tags=deepcopy(cmd.tags)):
                     self.send([self._add_uncompute_tag(cmd.get_inverse())])
 
             else:
-                if len(new_local_id) == 0:
-                    # No local qubits are active currently -> do standard
-                    # uncompute
-                    self.send([self._add_uncompute_tag(cmd.get_inverse())])
-                else:
-                    # Process commands by replacing each local qubit from
-                    # compute section with new local qubit from the uncompute
-                    # section
```

Comment:
I would suggest to keep a similar comment in order that to understand what this `else` branch is supposed to do